openerp-atomfeed-service/src/test/java/org/bahmni/feed/openerp/worker/OpenERPSellableResourceWorkerTest.java

```diff
@@ -0,0 +1,63 @@
+package org.bahmni.feed.openerp.worker;
+
+import org.bahmni.feed.openerp.OpenERPAtomFeedProperties;
+import org.bahmni.feed.openerp.client.OpenMRSWebClient;
+import org.bahmni.openerp.web.client.OpenERPClient;
+import org.bahmni.openerp.web.request.OpenERPRequest;
+import org.bahmni.openerp.web.request.builder.Parameter;
+import org.ict4h.atomfeed.client.domain.Event;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.net.URI;
+import java.util.Date;
+import java.util.List;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class OpenERPSellableResourceWorkerTest {
+
+    @Mock
+    private OpenERPAtomFeedProperties atomFeedProperties;
+    @Mock
+    private OpenERPClient openERPClient;
+    @Mock
+    private OpenMRSWebClient webClient;
+
+    private String feedUri = "http://feeduri";
+    private String resourceJson;
+    private OpenERPSellableResourceWorker worker;
+
+    @Before
+    public void setUp() {
+        resourceJson = "{\n" +
```

Comment:
Please use a representative json. We also want to test, if all the properties are getting serialized and bound property. For example, you have said bind id=>uuid, isActive => active, properties => properties etc. whether the dates are getting appropriately deserialized etc.
Example: try with this payload 

{
  "isActive": true,
  "dateCreated": "2015-09-07T12:14:24.000+0530",
  "lastUpdated": "2015-09-07T14:18:15.000+0530",
  "name": "Dressing of Wound",
  "id": "e48dd85b-fb13-4f3a-ae01-e6fd57eea6fe",
  "properties": {
    "sellable": "true"
  }
}